mods/ra/maps/allies-05a/weapons.yaml

```diff
@@ -2,19 +2,14 @@ MissionColt:
 	ReloadDelay: 6
 	Range: 7c0
 	Report: gun5.aud
-	Projectile: Bullet
-		High: true
-		Speed: 1c682
+	Projectile: InstantHit
 	Warhead@1Dam: SpreadDamage
 		Spread: 42
 		Damage: 50
 		Versus:
-			None: 0
 			Wood: 10
-			Light: 0
-			Heavy: 0
-			Concrete: 0
 		DamageTypes: Prone50Percent, TriggerProne, BulletDeath
+		AffectsParent: true
 	Warhead@2Eff: CreateEffect
 		Explosions: piffs
 		InvalidImpactTypes: Water
```

Comment:
Can you please add the following changes as well? The piffs and the decreasing health bar always irked me when playing this mission.

```diff
diff --git a/mods/ra/maps/allies-05a/weapons.yaml b/mods/ra/maps/allies-05a/weap
index 13e352d..5dfceef 100644
--- a/mods/ra/maps/allies-05a/weapons.yaml
+++ b/mods/ra/maps/allies-05a/weapons.yaml
@@ -7,9 +7,6 @@ MissionColt:
                Spread: 42
                Damage: 50
                Versus:
-                       Wood: 10
+                       Wood: 0
                DamageTypes: Prone50Percent, TriggerProne, BulletDeath
                AffectsParent: true
-       Warhead@2Eff: CreateEffect
-               Explosions: piffs
-               InvalidImpactTypes: Water
```